### ISSUE TITLE
Unable to scroll in source pane after chunk grading

### ISSUE THREAD

**Issue by rich-rstudio** [rstudio/rstudio#8008]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.4.915
    OS Version      : MacOS
    R Version       : 4.0.2

### Steps to reproduce the problem
1. register a callback
remotes::install_github("rstudio/rstudioapi")
rstudioapi::registerChunkCallback(mycallback)

```
mycallback <- function(chunkName, chunkCode) {
  template <- "<div id=\"abc123\">\n  <div class=\"callbackActions\">\n\n  </div>\n  <div class=\"callbackResult\">\n\n  </div>\n</div>\n\n<style>\n.callbackSuccess {\n  background-color: #ACA;\n}\n\n.callbackFailure {\n  background-color: #CAA;\n}\n</style>\n<script>\n  function handle(name, chunkCode, result){\n\n    var divID = \"abc123\"; // A random value generated in the R callback\n    var el = document.querySelectorAll('#' + divID)[0];\n    var resultsEl = el.querySelectorAll('.callbackResult')[0];\n    var actionsEl = el.querySelectorAll('.callbackActions')[0];\n\n    window.doGrade = function(url){\n      var request = new XMLHttpRequest();\n      request.open('POST', url, true); // Simulate a happy response\n      // request.open('GET', 'http://httpbin.org/post', true); // Simulate a bad response\n      // request.open('GET', 'http://yoidontevenexisthere.org/post', true); // Simulate an invalid request\n      request.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded; charset=UTF-8');\n  \n      request.onload = function() {\n        if (this.status >= 200 && this.status < 400) {\n          // Success!\n          var resp = this.response;\n          resultsEl.innerHTML = '<h3>Noice!</h3><p><pre>' + \n            'I ran the chunk `' + name + '` with the following code:\\n\\n```\\n' +\n            chunkCode + '\\n```\\n\\n and it all worked great!' +\n            '</pre></p><p><img src=\"https://www.unilad.co.uk/wp-content/uploads/2018/07/success-kid-original.jpg\" width=\"500px\" height=\"330px\"/></p>';\n          resultsEl.classList.add('callbackSuccess');\n          resultsEl.classList.remove('callbackFailure');\n        } else {\n        console.log(\"Setting\");\n        console.log(resultsEl);\n          // We reached our target server, but it returned an error\n          resultsEl.innerHTML = '<h3>Bad response</h3><p><pre>' + this.status + ' HTTP status</pre></p><p><img src=\"https://sayingimages.com/wp-content/uploads/so-sad-cat-meme.jpg\" /></p>';\n          resultsEl.classList.add('callbackFailure');\n          resultsEl.classList.remove('callbackSuccess');\n        }\n      };\n  \n      request.onerror = function() {\n        resultsEl.innerHTML = '<h3>I can\\'t even..</h3><p><img src=\"https://i.ytimg.com/vi/GTJQ3JsaHcA/hqdefault.jpg\" /></p>';\n        // There was a connection error of some sort\n        resultsEl.classList.add('callbackFailure');\n        resultsEl.classList.remove('callbackSuccess');\n      };\n\n      request.send('user=person&pwd=password&organization=place&requiredkey=key');\n    }\n      \n    actionsEl.innerHTML = '';\n    resultsEl.innerHTML = '';\n\n    if (name === 'nobuttons') {\n      // Don't add anything\n    } else { \n      actionsEl.innerHTML = '<button onclick=\"doGrade(\\'http://httpbin.org/post\\')\">Success</button>' +\n        '<button onclick=\"doGrade(\\'http://httpbin.org/get\\')\">Failure</button>' + \n        '<button onclick=\"doGrade(\\'http://thiserverdoesntevenexisthereyo.co.uk\\')\">Wat</button>';\n    }\n  }\n  handle(\"chunkName\", \"my code here\", {});\n</script>"
  rand <- round(runif(1, 1e8, 1e9))
  htmlres <- gsub("chunkName", chunkName, template)
  htmlres <- gsub("my code here", chunkCode, htmlres)
  return(list(html = htmlres))
}
```

2. Run a chunk in R notebook
3. Try to scroll in the source pane

### Describe the problem in detail
The user cannot scroll in the source pane after running the chunk. They have to move the mouse cursor to the scrollbar to scroll.
### Describe the behavior you expected
User should be able to scroll in the source pane

- [x ] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [ x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ x] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by jmcphers** [comment]
See also this very long-standing issue, which could be related (or the same bug): https://github.com/rstudio/rstudio/issues/2202.

**Comment by melissa-barca** [comment]
Hi @jmcphers looking into #2202 I'd say you're correct and this is the same issue. More rarely, there is extra white space in the callback output as well which is mentioned in the linked issue #6909

**Comment by kfeinauer** [comment]
Verified fixed in version 1.4.944
